infrastructure/main.tf

```diff
@@ -0,0 +1,27 @@
+resource "random_string" "password" {
+  length = 32
+  special = true
+}
+
+module "send-letter-service-api" {
+  source   = "git@github.com/hmcts/terraform-module-webapp.git"
+  product  = "${var.product}-api"
```

Comment:
we have not named the build as `send-letter-service-api` - it is only self-titled `send-letter-service` so only `product = "${var.product}"`

Reply:
updated the config